Requirements/ViewModels/RequirementBrowser/Rows/RequirementsSpecificationRowViewModel.cs

```diff
@@ -165,19 +166,23 @@ public void DragOver(IDropInfo dropInfo)
                 return;
             }
 
-            var spec = dropInfo.Payload as RequirementsSpecification;
+            //----------------------------------------------------------------------------------------------------------
+            // Drag a RequirementsSpecification onto another RequirementsSpecification??? What is the purpose of this?
+            //----------------------------------------------------------------------------------------------------------
```

Comment:
drag-n-drop a spec on another spec does not make sense. Please prohibit the app from doing that and do not leave commented code in the code-base

Reply:
Ok, will remove comments